environments/production/terraform.yml

```diff
@@ -367,6 +367,33 @@ proxy_servers:
     group: "proxy"
     os: ubuntu_pro_bionic
 
+  - server_name: "proxy_a001-production"
```

Comment:
I think our naming convention would suggest that we use either `proxy_a000-production` (adopt the naming we use for machines that use `count` in case we add that) or `proxy_a2-production` (increment 1 from the machines we're replacing). If you have another convention in mind this isn't a big deal, but I thought I would share.